src/Components/PreviousButton/PreviousButton.tsx

```diff
@@ -21,7 +21,7 @@ const PreviousButton = ({ navFunction }: Props) => {
 
   const defaultNavigate = () => {
     if (id && +id === householdStep + 1) {
-      navigate(`/${whiteLabel}/${uuid}/step-${householdStep}/${formData.householdData.length}`);
+      navigate(`/${whiteLabel}/${uuid}/step-${householdStep}/${formData.householdData.length || 1}`);
```

Comment:
Can we use `??`. I prefer it because it does not change behavior for `0` and `''`.
```suggestion
      navigate(`/${whiteLabel}/${uuid}/step-${householdStep}/${formData.householdData.length ?? 1}`);
```

Reply:
I used `||` intentionally to handle the 0 case. In the off chance the user goes back a page before the form data updates, the householdData length was 0, which caused a weird routing bug, so wanted it to default to 1 in that case.